server/src/test/java/org/opensearch/cluster/ClusterStateTests.java

```diff
@@ -86,31 +86,34 @@ public void testSupersedes() {
         final DiscoveryNode node2 = new DiscoveryNode("node2", buildNewFakeTransportAddress(), emptyMap(), emptySet(), version);
```

Comment:
Let's change the log [here](https://github.com/opensearch-project/OpenSearch/blob/737843fcc447c4e9167edc1e24a7ed59c043241c/server/src/test/java/org/opensearch/cluster/ClusterStateTests.java#L152) and the associated methods. Otherwise, it'll be difficult to track

Reply:
Are you referring to change the field name "master_node" ?
The link you pointed out is the actual response of Cluster State API, which is modified in the PR https://github.com/opensearch-project/OpenSearch/pull/2415 for version 2.0.
Both fields "master_node" and "cluster_manager_node" exist in the API response, and the reason to preserve the "master_node" field is to keep backwards compatibility.

Reply:
Thanks for the clarification. Makes sense